.github/workflows/create-release-pr.yaml

```
@@ -35,8 +60,8 @@ jobs:
         id: release-pr
         uses: changesets/action@aba318e9165b45b7948c60273e0b72fce0a64eb9 # v1.4.7
         with:
-          branch: 'main'
-          title: 'chore(release): bump version and update changelog'
-          commit: 'chore(release): bump version and update changelog'
+          branch: ${{ steps.branch-check.outputs.branch_name }} # This branch context is ignored in this step: `https://github.com/changesets/action/blob/50750fa876cc1e54c7cb972db5e2f7271fc53d99/src/run.ts#L328`, therefore the usage of `branch`
```

```suggestion
          # This branch context is ignored in this step: https://github.com/changesets/action/blob/50750fa876cc1e54c7cb972db5e2f7271fc53d99/src/run.ts#L328, therefore the usage of `branch`
          branch: ${{ steps.branch-check.outputs.branch_name }}
```
don't think the comment is really needed since it's obvious from the input. But if you want to add it, maybe add the comment before the statement so it's not a long line